src/pages/BooksPage/BooksPage.jsx

```diff
@@ -1,9 +1,111 @@
-import React from 'react'
+import React, { useEffect, useState } from "react";
+import {
+  useGetFictionQuery,
+  useGetNonfictionQuery,
+} from "../../api/books-api/books-api";
+import classes from "./BooksPage.module.css";
+import { Splide, SplideSlide } from "@splidejs/react-splide";
+import "@splidejs/splide/dist/css/splide.min.css";
+import { NavLink } from "react-router-dom";
 
 function BooksPage() {
+  const [maxSliderItems, setMaxSliderItems] = useState(5);
+
+  const { fiction } = useGetFictionQuery(undefined, {
+    selectFromResult: ({ data, ...rest }) => ({
+      fiction: data ? data.results.books : [],
+      data,
+      ...rest,
+    }),
+  });
+
+  const { nonfiction } = useGetNonfictionQuery(undefined, {
+    selectFromResult: ({ data, ...rest }) => ({
+      nonfiction: data ? data.results.books : [],
+      data,
+      ...rest,
+    }),
+  });
```

Comment:
You can move select from the result logic to rtk query configuration (to [src/api/books-api/books-api.js](https://github.com/KingaMiroslaw/my-books/pull/3/files#diff-6012f26f5af31d3d264a3c621e1cfc6592d54ad902a9af5508bd387a362568ba) )